service/api.go

```diff
@@ -345,25 +344,15 @@ func (as *APIServer) FetchExpansionsForTask(w http.ResponseWriter, r *http.Reque
 		as.LoggedError(w, r, http.StatusNotFound, errors.New("version not found"))
 		return
 	}
-	var u *user.DBUser
-	if t.ActivatedBy == evergreen.StepbackTaskActivator {
-		u, err = user.FindOneById(v.Author)
-	} else if t.ActivatedBy != "" {
-		u, err = user.FindOneById(t.ActivatedBy)
-	}
+
+	adminVars, err := projectVars.GetAdminOnlyVars(t)
```

Comment:
There's actually not really a point to having this be separate from GetUnrestrictedVars, since we end up looping twice, and when we get rid of restricted vars it can then just be a general "GetVars" funtion.

Reply:
added a comment to GetAdminOnly and GetUnrestricted also has a comment to be changed on 16045